backends/advanced/src/advanced_omi_backend/workers/transcription_jobs.py

```
@@ -222,11 +222,13 @@ async def transcribe_full_audio_job(
             sample_rate=16000,
             diarize=True,
         )
+    except ConnectionError as e:
+        logger.exception(f"Transcription service unreachable for {conversation_id}")
+        raise RuntimeError(str(e))
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)